pom.xml

```diff
@@ -542,6 +545,53 @@
                 <excludedGroups>org.wildfly.security.ExcludedOnIbmJdk</excludedGroups>
             </properties>
         </profile>
+        
+        <profile>
+            <id>methodSignatureGenerate</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.netbeans.tools</groupId>
+                        <artifactId>sigtest-maven-plugin</artifactId>
+                        <version>1.0</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>generate</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <packages>${public.packages}</packages>
+                        </configuration>
+                    </plugin>
+                </plugins>
+            </build>
+        </profile>
+        
+        <profile>
+            <id>methodSignatureCheck</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.netbeans.tools</groupId>
+                        <artifactId>sigtest-maven-plugin</artifactId>
+                        <version>1.0</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>check</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <packages>${public.packages}</packages>
+                            <releaseVersion>1.1.8.Final</releaseVersion>
```

Comment:
Is this meant to be the Elytron version? Could ${project.version} be used here instead?

Reply:
It is meant to be Elytron version, but some important version in history. In this case latest 1.1.x release which contains generated method signature file. There is no such release now. This is just guess.

${project.version} will resolve to something like 1.2.x.Final it is not what we want here.

Reply:
+1

Reply:
Makes sense. Just one more minor nit, the commit should reference an ELY issue.

Reply:
Looks like ELY-1398 - please reference it in commit message and in pull request title ;)
https://issues.jboss.org/browse/ELY-1398